### ISSUE TITLE
Does this action work for branches protected by merge queues?

### ISSUE THREAD

**Issue by ycheng517** [CasperWA/push-protected#257]

**Comment by CasperWA** [comment]
This is not tested, in theory it should work - it's a matter of testing and ensuring the required status checks are run.

**Comment by Horschig** [comment]
Yes it does. For the newer branch rulesets you need to add the correct bypass rules. For the old branch protection rules it works fine also (I only tried with `allow force push` for everyone there, despite using `force: false` for this action 🤷‍♂️).

**Comment by ycheng517** [comment]
awesome, thanks for the input
